src/GeneralStateTestsFiller/stEIP1559/typeTwoBerlinFiller.yml

```diff
@@ -0,0 +1,71 @@
+typeTwoBerlin:
+  _info:
+    comment: Ori Pomerantz qbzzt1@gmail.com
+
+  env:
+    currentCoinbase: 2adc25665018aa1fe0e6bc666dac8fc2697ff9ba
+    currentDifficulty: 0x20000
+    currentGasLimit: 0xFF112233445566
+    currentNumber: 1
+    currentTimestamp: 1000
+    previousHash: 5e20a0453cecd065ea59c37ac63e079ee08998b6045136a8ce6635c7912ec0b6
+#    currentBaseFee: 1000
```

Comment:
now either need to put in the state tests or refill with latest version of retesteth
because devs get confused the test is missing currentBaseFee in env section. 
(even for london) 
NOTE:  retesteth never send baseFee genesis params on Berlin and networks lower then London